CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT] Added etcd username and userpassword from config #4205
```

Comment:
Order of entries in the changelog should be: `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.

```suggestion
* [ENHANCEMENT] Etcd: Added username and password to etcd config. #4205
```

Reply:
@ymhhh Could you fix this, please? We would be very happy to merge once done! Thanks!